addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV6SessionManager.java

```diff
@@ -520,7 +520,7 @@ public void run() {
             }
         } catch (IOException e) {
             if (!willbeclosed) {
-                logger.error(e.getLocalizedMessage());
+                logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace